### DISCUSSION TITLE
Integration Kube-bench on RKE Cluster

### DISCUSSION THREAD

**Discussion by Mayurmaheshwari96** [aquasecurity/kube-bench#1177] [Questions and Help]
How can I integrate the Kube-bench with the Rancher in a way that if any cluster is created on that rancher then it automatically available on it and check it?

**Comment by mozillazg** [comment]
@Mayurmaheshwari96 Rancher has a built-in solution to integrate kube-bench. Here is the document: https://rancher.com/docs/rancher/v2.6/en/cis-scans/
